src/main/java/com/GeorgesServer/app/com/GeorgesServer/handler/FormHandler.java

```diff
@@ -21,6 +21,18 @@ public ServerResponse handle(ClientRequest clientRequest) {
           serverResponse = responseBuilder.getResponse();
       }
 
+      if (clientRequest.getMethod().equals("POST")) {
```

Comment:
I see the duplication, and It wont be here for long. Each of these conditionals will wind up doing calling different methods shortly.